catalog/video/video-mpegts-transformer/src/main/java/org/codice/alliance/transformer/video/MpegTsInputTransformer.java

```diff
@@ -136,11 +146,26 @@ public Metacard transform(InputStream inputStream, final String id)
 
             extractStanag4609Metadata(metacard, fileBackedOutputStream);
 
+            extractMediaEncodings(metacard, fileBackedOutputStream);
+
             return metacard;
         }
 
     }
 
+    private void extractMediaEncodings(Metacard metacard, TemporaryFileBackedOutputStream fbos)
+            throws IOException {
+
+        List<Serializable> serializables = TSStream.from(fbos.asByteSource())
+                .map(PESPacket::getStreamType)
+                .distinct()
+                .map(streamTypeToString)
+                .collect(Collectors.<Serializable>toList());
+
+        metacard.setAttribute(new AttributeImpl(AttributeNameConstants.MEDIA_ENCODING,
```

Comment:
You should be able to cast the List to Serializable : `metacard.setAttribute(AttributeNameConstants.MEDIA_ENCODING(AttributeNameConstants.MEDIA_ENCODING, (Serializable) serializables))`.  It is no longer true that all attributes will be multivalued.